tests/system/environmentLink_test.go

```diff
@@ -16,11 +18,11 @@ func TestEnvironmentLink(t *testing.T) {
 	t.Parallel()
 
 	s := NewSystemTest(t, "cases/environment_link", nil)
+	s.Terraform.Init()
```

Comment:
This should probably happen as part of `NewSystemTest` so callers don't accidentally forget.